src/android/SimpleCameraPreview.java

```diff
@@ -268,6 +235,16 @@ private boolean enable(JSONObject options, CallbackContext callbackContext) {
             e.printStackTrace();
         }
 
+        double aspectRatio = 4.0 / 3.0;
+        try {
+            if (options.getString("aspectRatio") != null && !options.getString("aspectRatio").equals("null")) {
+                aspectRatio = Double.parseDouble(options.getString("aspectRatio"));
```

Comment:
https://github.com/apache/cordova-android/blob/master/framework/src/org/apache/cordova/CordovaArgs.java

We can use getDouble check with @J4tinP